### DISCUSSION TITLE
is that framework suits for my work?

### DISCUSSION THREAD

**Discussion by ennteresan** [db0/godot-card-gaming#174] [Q&A] [answered]
Hi. I am searching best engine or framework for my game project.
I made some games with godot engine. And want to make card game like canasta or king .
Should i use this framework?

**Comment by db0** [comment, marked as answer]
Sure, assuming you are going to make your games open sourced, then this framework would fit

**Comment by ennteresan** [comment]
If i not want it open sourced , then it's problem?

16 Tem 2022 Cmt 12:52 tarihinde Divided by Zer0 ***@***.***>
şunu yazdı:

**Reply by db0** [reply]
Yes, this framework assumes share alike.

**Comment by GaminaeGaminae** [comment]
Do you think this could be used for multiplayer functionality?

**Comment by db0** [comment]
Yes, but you have to add the MP part yourself
